api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -150,28 +121,13 @@ public OfflineUserController(
             userService.syncEraCommonsStatusUsingImpersonation(user, Agent.asSystem());
 
         Timestamp newTime = updatedUser.getEraCommonsCompletionTime();
-        DataAccessLevel newLevel = user.getDataAccessLevelEnum();
-
-        if (!Objects.equals(newTime, oldTime)) {
-          log.info(
-              String.format(
-                  "eRA Commons completion changed for user %s. Old %s, new %s",
-                  user.getUsername(), oldTime, newTime));
-          changeCount++;
-        }
-        if (oldLevel != newLevel) {
-          log.info(
-              String.format(
-                  "Data access level changed for user %s. Old %s, new %s",
-                  user.getUsername(), oldLevel.toString(), newLevel.toString()));
-          accessLevelChangeCount++;
-        }
+        DataAccessLevel newLevel = updatedUser.getDataAccessLevelEnum();
```

Comment:
this change looks correct but it is a behavior change right?

Reply:
good catch!  Looks like we've been under-reporting this for a long time.

Reply:
(or maybe not, if the user gets updated behind the scenes?  hard to say)